### ISSUE TITLE
glibc2.14

### ISSUE THREAD

**Issue by Tjitskedv** [mozack/abra2#35]
I tried to run abra2-2.19.jar but get the following error:

Caused by: java.lang.UnsatisfiedLinkError:................./lib64/libc.so.
6: version `GLIBC_2.14' not found 

updating glibc without root access seem quite a hassle and isn't advised, so, is it possible to run abra without this version of glibc? Would it work if I build ABRA from source? I don't have experience with building java tools from source, so will only try that if I'm quite sure this is the solution, and when its possible without sudo rights. 

a previous version (ABRA0.96) did work on the system, but gave some memory issues which seem to be fixed in this new version.

Hope you can help

**Comment by mozack** [comment]
Difficult to say without access to your environment.  On what OS are you running?  Can you run docker or singularity?

**Comment by Tjitskedv** [comment]
Thank you for your response, I tried to run it on a cluster and don't know all the specs of that cluster. But It does work now, with help of the admin we were able to run it. As you suggested we made a singularity image and ran it with the command:

singularity exec abra-2.19.simg sh /run-abra2.sh [abra options]
or
singularity exec abra-2.19.simg java -Xmx16G -jar /abra2.jar [abra options]

first I ran into the error Exception in thread "main" java.nio.file.NoSuchFileException:
This was because the location of the file / temp dir wasn't bound to the singularity image, using a different location for the temp dir fixed this.
